models/solution/parsers/route_parser.rb

```diff
@@ -33,6 +33,7 @@ def self.parse(route, vrp, matrix, options = {})
                     @route.stops.any?(&:service_id)
 
       route_data ||= route_info(vrp)
+      route.stops.each{ |stop| stop.info.set_schedule(vrp, route.vehicle) }
```

Comment:
If you have this general function now, can't we remove the logic from the periodic heuristic as well? I do not see such removal in this PR, did I miss it?